api/src/main/java/ca/bc/gov/educ/grad/report/dao/GradDataConvertionBean.java

```diff
@@ -393,18 +405,23 @@ public List<Student> getStudents(ReportData reportData) {
         return result;
     }
 
-    public List<String> getCarrierPrograms(ReportData reportData) {
-        List<String> result = new ArrayList<>();
+    public GraduationStudentRecord getGraduationStudentRecord(ReportData reportData) {
         Student student = getStudent(reportData);
         String pen = student.getPen().getPen();
         GradSearchStudent gradSearchStudent = this.getStudentByPenFromStudentApi(pen, reportData.getAccessToken());
         if(gradSearchStudent != null) {
-            GraduationStudentRecord graduationStudentRecord = this.getGradStatusFromGradStudentApi(gradSearchStudent.getStudentID(), reportData.getAccessToken());
-            if(graduationStudentRecord != null) {
-                List<CareerProgram> careerPrograms = graduationStudentRecord.getCareerPrograms();
-                if (careerPrograms != null) {
-                    result.addAll(careerPrograms.stream().map(CareerProgram::getCareerProgramCode).collect(Collectors.toList()));
-                }
+            return getGradStatusFromGradStudentApi(gradSearchStudent.getStudentID(), reportData.getAccessToken());
+        }
+        return null;
+    }
+
+    public List<String> getCarrierPrograms(ReportData reportData) {
```

Comment:
@arybakov-cgi looks like there is a typo in the method name. getCarrierPrograms -> getCareerPrograms. Can you please take a look? Rest looks good.

Reply:
Thank You @arybakov-cgi  :-)